master/static/migrations/20220330212121_add_task_stats_table.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TYPE public.stats_type AS ENUM (
+    'QUEUED',
+    'IMAGEPULL'
+);
+
+CREATE TABLE public.task_stats (
+    allocation_id text NOT NULL REFERENCES public.allocations(allocation_id),
+    resource_pool text NOT NULL,
```

Comment:
I don't think this field is needed, because the referenced allocations table has a "resource_pool" column.

Reply:
I'd like to cache the resource pool name to avoid joining table after when aggregating

Reply:
hmm, won't this join will only happen when updating the aggregate tables? that's a rare periodic query. is this join that bad?

Reply:
After rethinking it, I think you are right, query only happens (on average) once a day, and we already have foreign key. Thank you for pointing this out, I've fixed the aggregate sql